src/main/scala/nl.knaw.dans.easy.deposit/docs/AgreementsXml.scala

```diff
@@ -21,8 +21,8 @@ import scala.util.Try
 import scala.xml.Elem
 
 object AgreementsXml extends SchemedXml {
-  override protected val schemaNameSpace = "http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
-  override protected val schemaLocation = "https://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2018/12/agreements.xsd"
+  val schemaNameSpace = "http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
```

Comment:
since these values are actually defined in `SchemedXml`, keep the `override`s.

Same for the other implementations of `SchemedXml`